Tools/ros2/ardupilot_msgs/srv/Takeoff.srv

```diff
@@ -0,0 +1,10 @@
+
+# This service requests the vehicle to takeoff
+
+# alt : Set the takeoff altitude
+
+uint8 alt
```

Comment:
Can you change this to a float, matching what the library supports? As a uint8, you can only takeoff to 255m.

Reply:
I tried with float but somehow service is not accepting given altitude with float and returning 0.00 as commanded altitude

Reply:
Ok. This is a major issue with the PR if we can only command integer resolution altitude and only 255m.

Is the DDS library not accepting the float? Some more information on the failure would be helpful to debug it.

Reply:
Yes it seems so. With float, the value that I am passing though ros service command is interpreted as 0.00. So `takeoff_request.alt` is 0.00 in any case.

Reply:
@Ryanf55 Any update regarding this float issue?

Reply:
I had no problems changing it to a float. Make sure you update both the `srv` and the `.idl` to match. 
![image](https://github.com/user-attachments/assets/4af3dc2f-3499-43c1-9048-3f2ed9e6019a)

Reply:
I made a PR to your repo. If you approve, please accept the changes and then fix the history.
https://github.com/snktshrma/ardupilot/pull/1

Reply:
I have merged the PR @Ryanf55. I got it working by cleaning the build and rebuilding. Thanks Ryan